packages/core/src/lib/createSystem.ts

```diff
@@ -102,6 +104,7 @@ export function createSystem(config: KeystoneConfig, isLiveReload?: boolean) {
           if (!isLiveReload) {
             await prismaClient.$connect();
             const context = createContext({ sudo: true });
+            await ensureSingletons(lists, prismaClient);
```

Comment:
This should not happen in production, we should automatically create a migration.

In development, we should _maybe_ throw if the constraints we rely on aren't being met, but, not in production.